src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -103,6 +106,13 @@ object HeaderPlugin extends AutoPlugin {
       ),
       headers.value,
       streams.value.log
+    ),
+    checkHeaders := checkHeadersTask(
+      FileFilter(Seq(excludes.value: _*)).filter(
+        (unmanagedSources in createHeaders).value.toList ++ (unmanagedResources in createHeaders).value.toList
+      ),
```

Comment:
This block is redundant with the call to `createHeaders` above. Is is possible to assign the result to a local variable and reuse it for both calls? Or will this mess up with the task execution of sbt?